SignalServiceKit/src/Messages/Attachments/TSAttachmentStream.m

```diff
@@ -695,9 +695,9 @@ - (nullable UIImage *)thumbnailImageWithThumbnailDimensionPoints:(NSUInteger)thu
 {
     OWSLoadedThumbnail *_Nullable loadedThumbnail;
     loadedThumbnail = [self loadedThumbnailWithThumbnailDimensionPoints:thumbnailDimensionPoints
-        success:^(OWSLoadedThumbnail *loadedThumbnail) {
```

Comment:
This change is cosmetic, just wanted to avoid shadowing the external variable to avoid similar problems.